data-manipulation/encryption/dpapi/encrypt-data-using-dpapi.yml

```diff
@@ -22,3 +22,10 @@ rule:
       - api: crypt32.CryptUnprotectData
       - api: System.Security.Cryptography.ProtectedData::Unprotect
       - api: System.Security.Cryptography.ProtectedData::Protect
+      - api: SystemFunction040
+      - api: SystemFunction041
```

Comment:
can you add a short description/comment on what these wrap?

Reply:
Thanks @mr-tz, I added comments in https://github.com/mandiant/capa-rules/pull/900/commits/eecaabe0bd038a029df4c3e65ccaabb502940206. Please let me know if that's fine.

Reply:
great, thanks